modules/compute/eks/README.md

```diff
@@ -112,7 +112,7 @@ EKS integrates with AWS Identity and Access Management (IAM) to control access t
 - `EksOidcArn`: The EKS Cluster's OIDC Arn
 - `EksClusterOpenIdConnectIssuer`: EKS Cluster's OPEN ID Issuer
 - `CNIMetricsHelperRoleName`: Name of role created for CNIMetricHelper SA
-- `EksClusterMasterRoleArn` - the masterrole used for cluster creation
+- `EksNodeRoleArn` - the role assigned to nodes when nodes are spinning up in node groups.
```

Comment:
Looks like you deleted `EksClusterMasterRoleArn` definition.....please put it back

Reply:
Good point. `EksClusterMasterRoleArn` seems to be removed. Pls add it back, as it is required by downstream modules to associate the OIDC provider.

Reply:
Sorry, this was accidentally removed somehow and I failed to catch it. Will put it back.